src/ctia/bundle/core.clj

```diff
@@ -74,9 +74,9 @@
       (transient-id? id) (assoc :original_id id)
       (seq filtered-ext-ids) (assoc :external_ids filtered-ext-ids))))
 
-(defn all-pages
+(s/defn all-pages
   "Retrieves all external ids using pagination."
-  [entity-type external-ids auth-identity]
```

Comment:
`read-store` in the body of this function was originally a var, and was ported to be an argument. However, due to the bad merge, the extra parameter was lost.